test/unit/foreman/renderer_test.rb

```diff
@@ -327,6 +327,56 @@ def setup_safemode_renderer
     assert @renderer.host_param_false?('false_param')
   end
 
+  test 'should have subnet_has_param?' do
+    @renderer.host = FactoryGirl.create(:host, :with_puppet)
+    subnet = FactoryGirl.create(:subnet_ipv4)
+    subnet.subnet_parameters.create(name: 'myparam', value: 'myvalue')
+    assert @renderer.subnet_has_param?(subnet, 'myparam').present?
+  end
+
+  test 'should have subnet_has_param? helper returning true' do
+    @renderer.host = FactoryGirl.create(:host, :with_puppet)
+    subnet = FactoryGirl.create(:subnet_ipv4)
+    subnet.subnet_parameters.create(name: 'myparam', value: 'myvalue')
+    assert @renderer.subnet_has_param?(subnet, 'myparam') == true
```

Comment:
no need for `== true`, just `assert @renderer.subnet_has_param?(subnet, 'myparam')`